superbench/benchmarks/return_code.py

```diff
@@ -18,7 +18,8 @@ class ReturnCode(Enum):
     MODEL_TRAIN_FAILURE = 11
     MODEL_INFERENCE_FAILURE = 12
     DISTRIBUTED_SETTING_INIT_FAILURE = 13
-    DATASET_GENERATION_FAILURE = 14
-    DATALOADER_INIT_FAILURE = 15
-    OPTIMIZER_CREATION_FAILURE = 16
-    MODEL_CREATION_FAILURE = 17
+    DISTRIBUTED_SETTING_DESTROY_FAILURE = 14
```

Comment:
Why not add it as 18 directly?

Reply:
Just put it near the DISTRIBUTED_SETTING_INIT_FAILURE error.